cnd/src/http_api/routes/index.rs

```diff
@@ -182,32 +190,66 @@ impl From<Body<HanEthereumEther, HalightLightningBitcoin>>
             ethereum_amount: body.alpha.amount,
             lightning_identity: body.beta.identity,
             lightning_cltv_expiry: body.beta.cltv_expiry.into(),
-            lightning_amount: body.beta.amount.0,
+            bitcoin_amount: body.beta.amount.0,
         }
     }
 }
 
 #[derive(serde::Deserialize, Clone, Debug)]
-pub struct HanEthereumEther {
+struct Hneth {
```

Comment:
```suggestion
struct HanEth {
```
This is my proposal re readability.

Reply:
Again agree with @D4nte here. 
Removing the ledger is a good idea though, because `HTLC for Asset on Native ledger` can imply the ledger I suppose. (because there is only one Native asset per ledger for that protocol if I am not mistaken.)

Reply:
> because there is only one Native asset per ledger for that protocol if I am not mistaken.

That is my understanding.